docs/user/collaboration.rst

```diff
@@ -0,0 +1,81 @@
+.. _collaborating:
+
+Collaborating on Renku
+======================
+
+Beyond building and maintaining your own data science workflows, there are many
+ways to use Renku to share your project with collaborators.
+
+Finding & accessing public projects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To find and access projects on the Renku platform that you might want to
+interact with, you can click the **Projects** link in the menu bar followed by
+the **Search** tab. Without entering any search terms, you will see a list of
+all public projects. You can click on the project names to be taken to the
+project's landing page.
+
+.. warning::
+
+  Note that the search box currently only allows you to search by project title,
+  not by username.
+
+By default, the files in public projects created by other people are
+"read only" to you. This means you can see an overview, create and read Kus,
+browse the filesystem in both text and :ref:`lineage` modes. The
+project's **Notebook Servers** tab will inform you that you do not have
+permissions to launch a notebook or make modifications to the code or datasets.
+
+To get access to the project, you can:
+
+* ask the project creator to `add you as a developer <added_to_project_>`_ or maintainer of the project through GitLab.
+
+If the project owner has not added you to the project, you can still participate:
+
+* through `Renku platform "Ku"s <make_kus_>`_, where you can post notes and discussion.
+* through `forking a project <forks_for_collaboration_>`_ through the Renku platform, which creates your own copy of the current state of the project that you own.
+
+.. _added_to_project:
+
+Add or be added as a developer on a project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To allow someone to make changes to your project, you can use the GitLab
+interface to add them in Settings > Members, with at either Developer or
+Maintainer privileges.
+
+Adding them here will make this project appear in their ``Your Projects`` list
+on the Renku platform, where the **Notebook Servers** tab for the project will allow
+them to launch an interactive environment to work on the project as usual.
+
+.. warning::
+
+  QUESTION: How are concurrent notebook session edits handled?
```

Comment:
what do you mean "concurrent"?

Reply:
concurrent: two people who have equal access to a project spawn notebooks. what happens when they try to modify files?

Reply:
their sessions are separate but the repository is shared - so a usual git workflow is recommended, e.g. using branches and merging to avoid conflicts as much as possible

Reply:
ok, cool! will add that to the docs as well!